src/BuildingRegistry/Legacy/Building.cs

```diff
@@ -23,12 +23,12 @@ public MigrateBuilding CreateMigrateCommand(Func<List<Legacy.BuildingUnit>, List
             var status = _status ?? throw new InvalidOperationException($"No status found for Building '{_persistentLocalId}'");
 
             return new MigrateBuilding(
-                _buildingId,
-                _persistentLocalId,
+                new BuildingId(new Guid("11111111-1111-1111-1111-111111111111")),
```

Comment:
what's this??

Reply:
Oei, ik moest van anais test scenario maken en had daarvoor speciale migratie moeten uitvoeren om een aggregate er in te steken met "isremoved == true" omdat we die niet in die state krijgen met commands.